### PR TITLE
[orc8r][1.4] Update orc8r installation for version 1.4

### PR DESCRIPTION
**Pull request by mpgermano**
Signed-off-by: Michael Germano <mgermano@fb.com>

## Summary

This PR serves to update the orc8r installation process for upcoming version 1.4.
The PR has a few related changes:
- Merge orc8r controller and orc8r base docker images into a single image. This is done to avoid having different images in prod and dev.
- Update terraform scripts to deploy orc8r service mesh helm charts. This includes up to 6 different helm charts based on what the `orc8r_deployment_type` variable is set to.
- Add `package.sh` script to handle all helm chart packaging and uploading. Previously this was a manual process laid out in the orchestrator installation documentation.

## Test Plan
Deployed to personal orc8r cluster

### CODE REVIEW

File: orc8r/cloud/deploy/terraform/orc8r-helm-aws/main.tf [hunk 1]
```diff
@@ -34,8 +34,155 @@ resource "helm_release" "orc8r" {
...
+  }
+
+  set_sensitive {
+    name  = "nms.magmalte.env.mysql_pass"
+    value = var.nms_db_pass
+  }
+}
+
+
+resource "helm_release" "fbinternal-orc8r" {
```
Comment (karthiksubraveti) [inline orc8r/cloud/deploy/terraform/orc8r-helm-aws/main.tf:132]: should we perhaps cleanup fbinternal and wifi pods going forward ?

**Review by karthiksubraveti — APPROVED** [review]
Changes LGTM. Thanks for adding the helm packaging script. It should simplify the usage for the users. Do you plan to add a readme for the script later during documentation ?

File: orc8r/cloud/deploy/terraform/orc8r-helm-aws/main.tf (hunk 1 above)
Comment (mpgermano) [reply orc8r/cloud/deploy/terraform/orc8r-helm-aws/main.tf:132]: These modules still provide services that we need  - though I think the wifi one may be able to be removed soon.

**Comment by mpgermano** [comment]
> Changes LGTM. Thanks for adding the helm packaging script. It should simplify the usage for the users. Do you plan to add a readme for the script later during documentation ?

Yeah I will add documentation to the "Build Orchestrator" doc that explains how this script should be used. In the future we should look at having the charts and images hosted by FFi to remove the build step entirely.

**Pull request merged**